integration/ct_integration_test.sh

```diff
@@ -3,6 +3,11 @@ set -e
 INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
 . "${INTEGRATION_DIR}"/common.sh
 
+echo "Building code"
```

Comment:
I'd prefer to keep the same quote style for all of the `echo`s, whether they do or do not happen to mention a variable.